docs/run.md

```diff
@@ -10,14 +10,26 @@ To use this module, you need to have a contract compiled.  So, if there is a con
 Environment
 ---------
 
--   `JavaScript VM`: All the transactions will be executed in
-    a sandbox blockchain in the browser. This means nothing
-    will be persisted when you reload the page. The JsVM is its own blockchain and on each reload it will start a new blockchain, the old one will not be saved.
+-   `Remix VM (London)` : For connecting to a sandbox blockchain in the browser. The Remix VM is its own "blockchain" and on each reload the old chain will be cleared and a new blockchain will be started. **The old one will not be saved**.  The London refers to the London fork of Ethereum.
```

Comment:
It will be better to add that `Remix VM was previously known as JavaScript VM on Remix IDE`